packages/@ember/-internals/glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2537,7 +2537,7 @@ moduleFor(
         template: '<div id="inner-value">{{this.wrapper.content}}</div>',
       });
 
-      let expectedBacktrackingMessage = backtrackingMessageFor('content', 'Wrapper', {
+      let expectedBacktrackingMessage = backtrackingMessageFor('content', '_?Wrapper', {
```

Comment:
This is interpreted as a regex, and when testing production builds under rollup we see the class name `_Wrapper`.